.github/workflows/run-link-check.yml

```diff
@@ -30,4 +30,4 @@ jobs:
           key: checkstyle-maven-cache-${{ hashFiles('**/pom.xml') }}
 
       - name: Check links
-        run: ./.ci/run-link-check-plugin.sh
+        run: "./.ci/run-link-check-plugin.sh --check-external"
```

Comment:
> We should only check for internal links in PRs.

Isn't this checking both internal and external?

Reply:
Yes, that is correct. We have some redundancy here between the github action and circleci.

Reply:
There might be some confusion on name of argument, is it what in addition or do-only.

If there better name please share how to name it.

Reply:
I am confused then, but not on argument name.

https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20%22run-link-check-plugin%22&type=code
Why does GH get lucky and tests both but circleci doesn't?

Reply:
GitHub action runs in master on both internal and external, for PRs we are only checking internal, for stability reasons.

Reply:
Should we document that, especially on circle?

Reply:
Code should be self explanatory.
And now I see a way to improve.
Action should do: `./.ci/run-link-check-plugin.sh`
Circleci: `./.ci/run-link-check-plugin.sh --skip-external`

Reply:
Done